tests/components/rfxtrx/test_sensor.py

```diff
@@ -118,59 +113,61 @@ async def test_discover_sensor(hass, rfxtrx):
     )
     await hass.async_block_till_done()
 
-    event = rfxtrx_core.get_rfx_object("0a520801070100b81b0279")
-    event.data = bytearray(b"\nR\x08\x01\x07\x01\x00\xb8\x1b\x02y")
-    await _signal_event(hass, event)
-
-    entity = rfxtrx_core.RFX_DEVICES["sensor_07_01"]["Temperature"]
-    assert 1 == len(rfxtrx_core.RFX_DEVICES)
-    assert {
-        "Humidity status": "normal",
-        "Temperature": 18.4,
-        "Rssi numeric": 7,
-        "Humidity": 27,
-        "Battery numeric": 9,
-        "Humidity status numeric": 2,
-    } == entity.device_state_attributes
-    assert "0a520801070100b81b0279" == entity.__str__()
-
-    await _signal_event(hass, event)
-    assert 1 == len(rfxtrx_core.RFX_DEVICES)
-
-    event = rfxtrx_core.get_rfx_object("0a52080405020095240279")
-    event.data = bytearray(b"\nR\x08\x04\x05\x02\x00\x95$\x02y")
-    await _signal_event(hass, event)
-    entity = rfxtrx_core.RFX_DEVICES["sensor_05_02"]["Temperature"]
-    assert 2 == len(rfxtrx_core.RFX_DEVICES)
-    assert {
-        "Humidity status": "normal",
-        "Temperature": 14.9,
-        "Rssi numeric": 7,
-        "Humidity": 36,
-        "Battery numeric": 9,
-        "Humidity status numeric": 2,
-    } == entity.device_state_attributes
-    assert "0a52080405020095240279" == entity.__str__()
-
-    event = rfxtrx_core.get_rfx_object("0a52085e070100b31b0279")
-    event.data = bytearray(b"\nR\x08^\x07\x01\x00\xb3\x1b\x02y")
-    await _signal_event(hass, event)
-    entity = rfxtrx_core.RFX_DEVICES["sensor_07_01"]["Temperature"]
-    assert 2 == len(rfxtrx_core.RFX_DEVICES)
-    assert {
-        "Humidity status": "normal",
-        "Temperature": 17.9,
-        "Rssi numeric": 7,
-        "Humidity": 27,
-        "Battery numeric": 9,
-        "Humidity status numeric": 2,
-    } == entity.device_state_attributes
-    assert "0a520801070100b81b0279" == entity.__str__()
-
-    # trying to add a switch
-    event = rfxtrx_core.get_rfx_object("0b1100cd0213c7f210010f70")
-    await _signal_event(hass, event)
-    assert 2 == len(rfxtrx_core.RFX_DEVICES)
+    await _signal_event(hass, "0a520801070100b81b0279")
+    state = hass.states.get("sensor.0a520801070100b81b0279_temperature")
+    assert state
+    assert state.state == "18.4"
+    assert (
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5802827)